### ISSUE TITLE
Tetrad manual

### ISSUE THREAD

**Issue by Mingyang-Cai** [cmu-phil/tetrad#1730]
I just started learning how to use Tetrad. However, I  can‘t open the web page. Could anyone help me solve this problem?

**Comment by jdramsey** [comment]
Hi @Mingyang-Cai, which web page are you trying to open? Maybe I can help.

**Comment by Mingyang-Cai** [comment]
Hi Joesph,
Thanks for you response. I can not open the Tetrad manual on this web page:https://www.ccd.pitt.edu/tools/.  I also attach a screenshot of the Manual button on this web page.
[https://s0.wp.com/i/blank.jpg]<https://www.ccd.pitt.edu/tools/>
Tools<https://www.ccd.pitt.edu/tools/>
We have written made available software and libraries that implement well known and experimental causal discovery algorithms: Notes: Bugs and Issues: Please report any bugs that you m…
www.ccd.pitt.edu
[cid:e98977c6-7dca-418d-bf06-5454ad49c795]

Best wishes,
Mingyang

________________________________

**Comment by cg09** [comment]
The tutorial link on the center for causal discovery website does not work.

Clark

**Comment by jdramsey** [comment]
Oh wow, it seems that that method of viewing HTML documents in a GitHub repository is not working. Maybe it's temporarily down. Let's wait a bit to see if it comes back up. Otherwise, I'll have to come up with a new way to reference that material.

**Comment by jdramsey** [comment]
No, I think I know the answer! It moved! Let me get an updated link for you, and then I can request to have these links updated on those websites.

**Comment by jdramsey** [comment]
Here's the new link:

https://htmlpreview.github.io/?https:///github.com/cmu-phil/tetrad/blob/development/tetrad-lib/src/main/resources/docs/manual/index.html

I'll also need to update the link Clark mentioned. I can't do these updates myself, but I'll put in the requests.

Thanks so much, I would never have caught that!

**Comment by jdramsey** [comment]
And here is the updated link to the other page:

https://htmlpreview.github.io/?https://github.com/cmu-phil/tetrad/blob/development/tetrad-lib/src/main/resources/docs/javahelp/manual/tetrad_tutorial.html

**Comment by jdramsey** [comment]
Hopefully, these will be updated on the CCD page soon. I also requested they be updated on the Tetrad web page.

**Comment by Mingyang-Cai** [comment]
@jdramsey Thanks so much for your help.

**Comment by jdramsey** [comment]
Absolutely!
